### ISSUE TITLE
t/defer.t fails (with older File::ShareDir?)

### ISSUE THREAD

**Issue by eserte** [robrwo/HTML-DeferableCSS#1]
On a few of my smokers the tests are failing:
```
Name "File::ShareDir::MODULE_SHARE" used only once: possible typo at t/defer.t line 7.
File 'cssrelpreload.min.js' does not exist in module dir at /home/cpansand/.cpan/build/2020021910/HTML-DeferableCSS-v0.1.1-wK_qNH/blib/lib/HTML/DeferableCSS.pm line 139.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 2 just after 2.
t/defer.t .................... 
Dubious, test returned 2 (wstat 512, 0x200)
All 2 subtests passed 
```
Statistical analysis suggests that this happens if an older version of File::ShareDir is installed:
```
****************************************************************
Regression 'mod:File::ShareDir'
****************************************************************
Name                   Theta          StdErr     T-stat
[0='const']           0.0000          0.0000      25.62
[1='eq_1.104']       -0.0000          0.0000     -13.69
[2='eq_1.116']        1.0000          0.0000    44585907624129888.00

R^2= 1.000, N= 77, K= 3
****************************************************************
```

**Comment by robrwo** [comment]
Thanks. I've tried to fix this in v1.2 https://metacpan.org/release/RRWO/HTML-DeferableCSS-v0.1.2 let me know if that fixes it.

**Comment by eserte** [comment]
Confirmed, 0.1.2 looks fine.
